genprotocol/sserver-avdl/sserver.avdl

```diff
@@ -0,0 +1,7 @@
+@namespace("sserver")
+protocol searchServer {
+  void writeIndex(bytes secureIndex, string docID);
```

Comment:
Might be nice to have a proper type for doc IDs, typedef'd to `string` or `int` or `bytes`, depending on the type of nonce you use.  (`string` probably isn't right for a string that contains encrypted data, for example.)